src/ms.tex

```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
 % Transition of needs: From new features, to community development, sustainability, ...
 % - pipeline between User, Participant, Code contributor, Maintainer, Coordinator.
 
+\subsection{Inclusion, Diversity, and Equity Programs} \label{sec:project-ide}
+
+\secauthor{Lía Corrales}
```

Comment:
@eblur : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

Reply:
Okay. I would appreciate one other person to help with this section.